docs/readthedocs/io/csvfiles.md

```diff
@@ -24,80 +24,180 @@ You may extend / alter the naming with pre- or suffix by calling `new EntityPers
+   * - schematic line graphic
+     - *prefix_* line_graphic_input *_suffix*
 
+```
 
 ### Time Series
 
-| Model                  | File Name                                 |
-|:-----------------------|:------------------------------------------|
-| individual time series | *prefix_* its *_columnScheme_UUID_suffix* |
-| load profile input     | *prefix_* rts *_profileKey_UUID_suffix*   |
-
+```{eval-rst}
+.. list-table::
+   :widths: 50 50
+   :header-rows: 0
+
+   * - Model
+     - File Name
+   * - individual time series
+     - *prefix_* its *_columnScheme_UUID_suffix*
+   * - load profile input
+     - *prefix_* rts *_profileKey_UUID_suffix*
+```
 
 Let's spend a few more words on the individual time series:
 Those files are meant to carry different types of content - one might give information about wholesale market prices,
 the other is a record of power values provided by a real system.
 To be able to understand, what's inside of the file, the *columnScheme* part of the file name gives insight of its
 content.
-The following keys are supported until now:
 
-| Key     | Information and supported head line                                                                                                                      |
-|:--------|:---------------------------------------------------------------------------------------------------------------------------------------------------------|
-| c       | An energy price (e.g. in €/MWh; c stands for charge). <br> Permissible head line: ``uuid,time,price``                                                    |
-| p       | Active power <br> Permissible head line: ``uuid,time,p``                                                                                                 |
-| pq      | Active and reactive power <br> Permissible head line: ``uuid,time,p,q``                                                                                  |
-| h       | Heat power demand <br> Permissible head line: ``uuid,time,h``                                                                                            |
-| ph      | Active and heat power <br> Permissible head line: ``uuid,time,p,h``                                                                                      |
-| pqh     | Active, reactive and heat power <br> Permissible head line: ``uuid,time,p,q,h``                                                                          |
-| weather | Weather information <br> Permissible head line: ``uuid,time,coordinate,direct_irradiation,diffuse_irradiation,temperature,wind_velocity,wind_direction`` |
+For example, you have an IndividualTimeSeries CSV file for energy prices, then you use the key `c` from the table below
+for columnScheme `its_c_2fcb3e53-b94a-4b96-bea4-c469e499f1a1.csv`.
+The CSV file must then have the appropriate format for the key `c` :
+
+```{eval-rst}
+.. list-table::
+   :widths: auto
+   :header-rows: 0
+   
+   * - ``time,price``
+   * - 2020-01-01T00:00:00Z;100.0
```

Comment:
Ok, sorry. I took a closer look and we're talking about csv files, so maybe some format closer to what you did before is better. Sorry.

Maybe using triple accents like this would make it look better?

````
```
time,price
2020-01-01T00:00:00Z,100.0
```
````

Reply:
Or rather, using [code blocks](https://myst-parser.readthedocs.io/en/v0.15.1/syntax/syntax.html#parameterizing-directives)?

Reply:
Is it desired that the date is highlighted?

Reply:
Let me check this once more...

Reply:
@pierrepetersmeier found a way to disable the highlighting.